src/vm/fieldmarshaler.cpp

```diff
@@ -435,7 +435,9 @@ VOID ParseNativeType(Module*                     pModule,
             {
                 if (fDefault || ntype == NATIVE_TYPE_STRUCT)
                 {
-                    selectedNft = InitFieldMarshaler<FieldMarshaler_Decimal>(pfwalk->m_FieldMarshaler, NATIVE_FIELD_CATEGORY_NESTED_VALUE_CLASS);
+                    // The decimal type can't be blittable since the managed and native alignment requirements differ.
```

Comment:
But what about when it is the only field in a struct?

Reply:
It changes the struct's alignment, so any struct containing that struct and something else might have layout differences. We'd basically have to track this throughout the type system to verify that we don't accidentally mark something blittable that isn't.